internal/winapi/cimfs.go

```diff
@@ -0,0 +1,45 @@
+package winapi
+
+import (
+	"syscall"
+	"unsafe"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+)
+
+type g = guid.GUID
```

Comment:
Why this?

Reply:
It seems that the mksyscall command won't import some packages (like this `guid` package) unless they are used somewhere in this file. So if we don't do this the generated go file complains about `unknown type guid`. But maybe there is a better way to fix this that I don't know.